apps/backend-functions/src/movie.ts

```diff
@@ -140,7 +140,7 @@ export async function onMovieUpdate(change: BlockframesChange<Movie>) {
     const movieWasSubmittedOn = wasSubmittedOn(before.app, after.app)[0];
     // Mail to supportEmails.[app]
     const from = getMailSender(movieWasSubmittedOn);
-    await sendMail(sendMovieSubmittedEmail(movieWasSubmittedOn, after), from, groupIds.noUnsubscribeLink);
+    await sendMail(await sendMovieSubmittedEmail(movieWasSubmittedOn, after, db), from, groupIds.noUnsubscribeLink);
```

Comment:
you already have the organizations here 
`const organizations = await getOrganizationsOfMovie(after.id);`
```suggestion
    await sendMail(sendMovieSubmittedEmail(movieWasSubmittedOn, after, organizations[0]), from, groupIds.noUnsubscribeLink);
```

Reply:
I  have to keep the 'await' without it i get an error, I find it litle strange

Reply:
remove async here `export async function sendMovieSubmittedEmai`